docs/tutorial/part-two/index.md

```diff
@@ -3,91 +3,45 @@ title: Introduction to Using CSS in Gatsby
-
-Typography.js is a JavaScript library which generates typographic CSS.
-
-Instead of directly setting the `font-size` of different HTML elements, you tell
-Typography.js things like your desired `baseFontSize` and `baseLineHeight` and
-based on these, it generates the base CSS for all your elements.
-
-This makes it trivial to change the font size of all elements on a site without
-having to directly modify the dozens of CSS rules.
-
-Using it looks something like this:
-
-```javascript
-import Typography from "typography"
-
-const typography = new Typography({
-  baseFontSize: "18px",
-  baseLineHeight: 1.45,
-  headerFontFamily: [
-    "Avenir Next",
-    "Helvetica Neue",
-    "Segoe UI",
-    "Helvetica",
-    "Arial",
-    "sans-serif",
-  ],
-  bodyFontFamily: ["Georgia", "serif"],
-})
-```
-
-## Gatsby plugins
-
-But before you can get back to building and trying out Typography.js, let's make
-a quick diversion and talk about Gatsby plugins.
-
-You're probably familiar with the idea of plugins. Many software systems support
-adding custom plugins to add new functionality or even modify the core workings
-of the software.
-
-Gatsby plugins work the same way.
-
-Community members (like you!) can contribute plugins (small amounts of
-JavaScript code) that others can then use when building Gatsby sites.
+### Creating global styles with standard CSS files
 
-There are already hundreds of plugins! Check them out at the
-[plugins section of the site](/plugins/).
+One of the most straightforward ways to add global styles to a site is using a global `.css` stylesheet.
 
-Our goal with Gatsby plugins is to make them straightforward to install and use. In almost every Gatsby site you
-build, you will be installing plugins. While working through the rest of the
-tutorial, you'll have many opportunities to practice installing and using
-plugins.
+#### ✋ Create a new Gatsby site
 
-## Installing your first Gatsby plugin
+Start by creating a new Gatsby site. It may be best (especially if you're new to the command line) to close the terminal windows you used for [part one](/tutorial/part-one/), and start a new terminal session for part two.
 
-Start by creating a new site. At this point, it probably makes sense to close the terminal windows you used to build tutorial-part-one so that you don't accidentally start building tutorial-part-two in the wrong place. If you don't close tutorial-part-one prior to building tutorial-part-two, you will see that tutorial-part-two appears at localhost:8001 instead of localhost:8000.
-
-Just like in part one, open a new terminal window and run the following commands to create a new Gatsby site in a directory called `tutorial-part-two`. Then, change to this new directory:
+Open a new terminal window, create a new "hello world" gatsby site, and start the development server:
 
 ```shell
 gatsby new tutorial-part-two https://github.com/gatsbyjs/gatsby-starter-hello-world
 cd tutorial-part-two
+gatsby develop
```

Comment:
I don't think we want to start it just yet, do we? They'll need to restart for `gatsby-browser.js` to pick up the changes.

Reply:
So... I also thought that was the case, but I tested it, and it doesn't need a restart.